clientgen/client.go

```diff
@@ -47,6 +49,12 @@ func (p *Plugin) MutateConfig(cfg *config.Config) error {
 		return xerrors.Errorf("parse query document failed: %w", err)
 	}
 
+	for i, queryDocument := range queryDocuments {
```

Comment:
In this layer, I want to use a function that represents a use case, so I want to separate it by a function.

Reply:
Use case as in the CleanCode architecture? So you want me to create a function for it?

Reply:
maybe yes. I want you to make i

Reply:
I looks like a problem what duplicate struct name when there are multiple Unnamed queries. Do you have this solution ?